CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ Release report: TBD
 
 ### Added
 
+- Add tests for msu patches with no associated CVE . ([#4009](https://github.com/wazuh/wazuh-qa/pull/4009)) \- (Framework + tests)
```

Comment:
```suggestion
- Add tests for msu patches with no associated CVE . ([#4009](https://github.com/wazuh/wazuh-qa/pull/4009)) \- (Framework + Tests)
```

Reply:
done in https://github.com/wazuh/wazuh-qa/pull/4009/commits/2c11167dfbd10f2e4f190449c19831b52cde646a